test/meson.build

```diff
@@ -18,11 +18,11 @@ tests = [
     'decoderstreamreader',
     'rawstreamreader',
     'bufferstreamer',
-    'parseLongPath'
+    'parseLongPath' 
```

Comment:
Trailing space. I have seen other I believe... editor should be configured to mark them, so you can easiky see them and delete them.